CRM-PricingBooks/Database/Models/ProductPrice.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace CRM_PricingBooks.Database.Models
+{
+    public class ProductPrice
+    {
+        public string ProductCode{ get; set; }
+
+        public double FixedPrice{ get; set; }
+    }    
```

Comment:
No debería tener un PromotionPrice también?

Reply:
Looking good!! But PromotionPrice is missing

Reply:
Looks great! But you need to add Promotion Price

Reply:
ProportionPrice has to be in DTO models not in database layer, because it is calculated at business logic layer